corehq/apps/userreports/pillow.py

```diff
@@ -57,9 +57,18 @@ def rebuild_table(self, table):
         table.rebuild_table()
 
     def python_filter(self, doc):
-        # filtering is done manually per indicator set change_transport
+        # filtering is done manually per indicator see change_transport
         return True
 
+    def change_trigger(self, changes_dict):
```

Comment:
@benrudolph there are 2 kinds of deletion. One is a real deletion where the doc is actually deleted from the database and the second is a 'fake' deletion where we just change the doc_type so that the doc doesn't get picked up by any of the views etc.

The second case is taken care of already but the first case was not: see https://github.com/dimagi/pillowtop/blob/master/pillowtop/listener.py#L328